spree/spree
# Allow user to add names to addresses.

Allows user to add an optional, but unique address name to each of their address.

### HTML CSS Improvements

- Moved the edit and delete address icons outside of the html label, no nested buttons in a selectable element.
- Added red border to fields with submit errors giving clear indication of errors.
- Now text says **New Address** and **Edit Address** was **New Shipping Address** and **Edit Shipping Address**.
### Tests Strengthened 
- User FFaker for randomised address data
- Check the cloned user_id to be_nil
- Check the cloned label to be_nil (Address names need to be unique, not cloned).
